conn2res/plotting.py

```diff
@@ -0,0 +1,248 @@
+# -*- coding: utf-8 -*-
+"""
+Plotting functions
+
+@author: Estefany Suarez
+"""
+
+import os
+import seaborn as sns
+import matplotlib.pyplot as plt
+import numpy as np
+from numpy.linalg import svd
+
+PROJ_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+FIG_DIR = os.path.join(PROJ_DIR, 'figs')
+if not os.path.isdir(FIG_DIR):
+    os.makedirs(FIG_DIR)
+
+
+class PCA:
```

Comment:
I created this PCA class to have a simple PCA object where we can specify which principal component we want to keep (see idx_pcs) and we can also decide to normalize the PCs by their singular values if we want so.

Perhaps it would be better to inherit this class from sklearn.decomposition.PCA but I thought it was quicker to implement my PCA from scratch than figuring out how sklearn.decomposition.PCA works. But I might have been wrong and looking backwards probably it would have been more principled to invest time understanding sklearn.decomposition.PCA properly and only extend its functionalities with a new class where needed.